### ISSUE TITLE
Can't unscheduled campaign

### ISSUE THREAD

**Issue by RudenkoDenis** [brandonseydel/MailChimp.Net#358]
Hi @brandonseydel 
I have  schedulle on campaign
I need unscheduled this campign
I try execute 
 **_mailChimpManager.Campaigns.ExecuteCampaignActionAsync(campaigId, CampaignAction.Unschedule).GetAwaiter().GetResult();**

![image](https://user-images.githubusercontent.com/24713107/48942118-c2280b00-ef26-11e8-82a7-23638aa0d6ea.png)

**but recivied error**
   в MailChimp.Net.Logic.CampaignLogic.<ExecuteCampaignActionAsync>d__13.MoveNext()
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   в MailCXhimp_3._0._0.Program.Main() в D:\WORK\Dev_Code\MailChimp\MailCXhimp-3.0.0\MailCXhimp-3.0.0\Program.cs:строка 66
   в System.AppDomain._nExecuteAssembly(RuntimeAssembly assembly, String[] args)
   в System.AppDomain.ExecuteAssembly(String assemblyFile, Evidence assemblySecurity, String[] args)
   в Microsoft.VisualStudio.HostingProcess.HostProc.RunUsersAssembly()
   в System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   в System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   в System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   в System.Threading.ThreadHelper.ThreadStart()

Tell me what wrong I did?
Thanks!

**Comment by RudenkoDenis** [comment]
@brandonseydel,
I updated on assembly 3.4.0 and used method UnscheduleAsync (helped https://github.com/brandonseydel/MailChimp.Net/issues/290)
Yes,  after execute 
_mailChimpManager.Campaigns.UnscheduleAsync(campaigId, new CampaignScheduleRequest()).GetAwaiter().GetResult();

  campaign has status "Pause", but when I execute 
var campaign1 = _mailChimpManager.Campaigns.GetAsync(campaigId).Result;
recieved error
Could not convert string to DateTime: -001-11-30T00:00:00+00:00. Path 'send_time', line 1, position 273.
![image](https://user-images.githubusercontent.com/24713107/48947830-3371b900-ef3b-11e8-939f-49d8c449790e.png)
Tell me what wrong I did?
Thanks!
